File: src/touch.c
#include <stdio.h>

int main(int argc, char** argv)
{
    if (argc<2)
    {
        printf("Error: need more arguments!\nUsage: touch \"File Name\"");
    }
    else{
        for (int i = 0; i < argc; ++i)
        {
            if (i>0)
            {
                FILE *fp;
                fp=fopen(argv[i],"w");
                fclose(fp);
            }
        }
    }
    return 0;
}

File: src/cs/touch.cs
using System;
using System.IO;

class touch
{
    static void Main(string[] args)
    {
        if (args.Length < 1)
        {
            Console.WriteLine("Error: need more arguments!\nUsage: touch \"File Name\"");
        }
        else
        {
            for (int i = 0; i<args.Length; i++)
            {
                if (File.Exists(args[i]) == true)
                {
                    Console.WriteLine("Error: file {0} exists!", args[i]);
                }
                else
                {
                    File.Create(args[i]);
                }
                
            }

        }
    }
}